src/elements/MenuDrawer/MainMenu/MainMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { Icon } from '@elements';
+import { IconName } from '@elements/Icon';
+import * as colors from '@util/colors';
+
+import styles from './MainMenu.styles';
+
+const MainMenu = props => {
+	let menuStyle = {};
+	const { icon, text } = props;
```

Comment:
not a big deal but you can save a line but destructuring the `icon` and `text` in place of `props` 
i.e. `const MainMenu = ({ icon, text }) => {`
also I think our linter prefers it

Reply:
Oh I didn't think of doing that! Good catch 👍